packages/paste-website/src/pages/primitives/listbox-primitive/index.mdx

```diff
@@ -0,0 +1,322 @@
+
+Because these are not styled, rendering any of them _as_ another component you can mix the functionality of two components together. Styling from one, listbox functionlity from the primitive component.
+
+In the example below, we import the Paste Button `import {Button} from '@twilio-paste/button';` and use it as the Listbox items via the `as` prop.
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+    Button,
+    Stack,
+    CheckboxCheckIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {customExample}
+</LivePreview>
+
+---
+
+## Usage Guide
+
+This package is a wrapper around the [Reakit Composite](https://reakit.io/docs/composite/). If you’re wondering why we wrapped that package into our own, we reasoned that it would be best for our consumers’ developer experience. For example:
+
+- If we want to migrate the underlying nuts and bolts in the future, Twilio products that depend on this primitive would need to replace all occurrences of `import … from ‘x-package’` to `import … from ‘@some-new/package’`. By wrapping it in `@twilio-paste/x-primitive`, this refactor can be avoided. The only change would be a version bump in the ‘package.json` file for the primitive.
+- We can more strictly enforce semver and backwards compatibility than some of our dependencies.
+- We can control when to provide an update and which versions we allow, to help reduce potential bugs our consumers may face.
+- We can control which APIs we expose. For example, we may chose to enable or disable usage of certain undocumented APIs.
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/listbox-primitve - or - yarn add @twilio-paste/core
+```
+
+#### Props
+
+This props list is a scoped version of the properties available from the [Reakit Composite](https://reakit.io/docs/composite/) package.
+
+##### useListboxPrimitiveState
+
+| Prop           | Type                         | Description                                            | Default     |
+| -------------- | ---------------------------- | ------------------------------------------------------ | ----------- |
+| `baseId?`      | `string`                     | ID that will serve as a base for all the items IDs     |             |
+| `rtl?`         | `boolean`                    | Determines how next and previous functions will behave |             |
+| `orientation?` | `"horizontal" \| "vertical"` | Defines the orientation of the listbox                 | "vertical"` |
+| `currentId?`   | `string`                     | The current focused item id                            |             |
+
+##### ListboxPrimitive
+
+**Note:** Most required props are provided by spreading the returned state from `useListboxPrimitiveState`.
```

Comment:
Question (non-blocking): Should this be a neutral callout?

Reply:
Not sure! Copied what was done in other primitives.